docs/rules/no-boolean-conditional.md

```diff
@@ -0,0 +1,28 @@
+# Disallow boolean literals in conditional expressions (no-boolean-conditional)
+
+It's a common mistake in JavaScript to use a conditional expression to select between two Boolean values instead of using ! to convert the test to a Boolean.
```

Comment:
This section needs to be longer. Show some examples and explain why they are bad practice.